src/components/FurnitureField/FurnitureField.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Seems like this file isn't being used anymore, if it isn't then go ahead and delete it.